contracts/apps/AragonApp.sol

```diff
@@ -5,21 +5,24 @@
 pragma solidity ^0.4.24;
 
 import "./AppStorage.sol";
-import "./IAragonApp.sol";
 import "../acl/ACLSyntaxSugar.sol";
 import "../common/Autopetrified.sol";
 import "../common/ConversionHelpers.sol";
 import "../common/ReentrancyGuard.sol";
 import "../common/VaultRecoverable.sol";
 import "../evmscript/EVMScriptRunner.sol";
+import "../lib/standards/ERC165.sol";
 
 
 // Contracts inheriting from AragonApp are, by default, immediately petrified upon deployment so
 // that they can never be initialized.
 // Unless overriden, this behaviour enforces those contracts to be usable only behind an AppProxy.
 // ReentrancyGuard, EVMScriptRunner, and ACLSyntaxSugar are not directly used by this contract, but
 // are included so that they are automatically usable by subclassing contracts
-contract AragonApp is IAragonApp, AppStorage, Autopetrified, VaultRecoverable, ReentrancyGuard, EVMScriptRunner, ACLSyntaxSugar {
+contract AragonApp is ERC165, AppStorage, Autopetrified, VaultRecoverable, ReentrancyGuard, EVMScriptRunner, ACLSyntaxSugar {
+    // Includes appId and kernel methods:
+    bytes4 internal constant ARAGON_APP_INTERFACE_ID = bytes4(0x54053e6c);
```

Comment:
Does this need to move to the implementation contract? I was thinking the interfaces should declare their expected constants, especially in the case of ERC165 identifiers.

Reply:
Sounds good to me. Actually, as you know, I would have everything in the interface, haha.